include/crm/lrmd.h

```diff
@@ -46,6 +51,8 @@ typedef struct lrmd_key_value_s {
  */
 #define LRMD_MIN_PROTOCOL_VERSION "1.0"
```

Comment:
We can stop using and deprecate this constant, and instead compare major versions

Reply:
It's coincidental that MIN is a .0 version, but it's a convenient coincidence. If we make it a macro, we don't have worry about ever having to update it when the main version changes.